.pre-commit-config.yaml

```
@@ -86,7 +86,7 @@ repos:
     - tomli
 
 - repo: https://github.com/astral-sh/ruff-pre-commit
-  rev: v0.1.1
+  rev: v0.1.2
   hooks:
   - id: ruff
     args: [--fix]
```

```suggestion
  rev: v0.1.2
  hooks:
  - id: ruff
    args: [--fix]
  - id: ruff-format
```
I forgot that the ruff formatter isn't enabled automatically so we have to specify it.  Thank you again!